setup.py

```diff
@@ -11,24 +9,20 @@
 
 
 requires = [
-    'requests',
+    'requests>=2.6.0',
 ]
 
-version = ''
-with open('telex/__init__.py', 'r') as fd:
```

Comment:
Nothing, it just seemed unnecessary. Could also import `__version__` explicitly I guess too.